style.css

```diff
@@ -0,0 +1,219 @@
+	width: 60px;
+  }
+.navbar_tablet{
+	display:none; 
+ }
+ header h2{
+	display:none;
+}
+/*** Big news section **/
+ .big-news{
+	max-width: 1200px;
+	margin: 0 auto;
+	display:flex;
+	flex-direction: column;
+	justify-content: center;
+	padding-top: 30px;
+ }
+ .news{
+	padding:3rem;
+  	width:480px;
+  	height:90%;
+
+  }
+  .news img { 
+	width:400px;
+  height:90%
+} 
+/*** News cards section **/
+.card {
+  padding: 3rem;
+  width:300px;
+  height:90%; 
+}
+
+.news_cards {
+	max-width: 1000px;
+	margin: 0 auto;
+	display: grid;
+	grid-gap: 3rem;
+	justify-content: center;	
+}
+.news_cards img { 
+	width:100%; 
+	height:100%; 
+	
+} 
+
+/*** Media Queries **/
+@media (max-width: 899px) and (min-width: 600px) {
+	.news_cards { 
+		grid-template-columns: repeat(2, 1fr); 
+		padding-right: 30 0px;
+	}
+	.navbar_tablet{
+		color:black;
+		text-decoration:none;
+		width:100%;
+		display:flex;
+		align-items: center;
+		justify-content: space-around;	
+	}
+	header h1{
+		display:none;
+	}
+	header h2{
+		display:none;
+	}
+	.big-news{
+		padding: 20px 5px 20 px 5px;
+		max-width: 1000px;
+		margin: 0 auto;
+		display:flex;
+		flex-direction: row;
+		justify-content: center;
+	 }
+	 .news{
+		padding: 1rem;
+		width:650px;
+		height:90%;	
+	  }	
+	 .icon{
+		display:none;
+	  }
+	  html { font-size: 16px; 
+	  }
+}
+@media (min-width: 900px) {
+	.news_cards { 
+		grid-template-columns: repeat(4, 1fr); 
+	}
+	nav{
+		color:black;
+		text-decoration:none;
+		width:50%;
+		display:flex;
+		justify-content: space-around;
+	}
+	header h1{
+		font-size: 40px;
+		margin:0px 30px 0px 10px;
+		padding:0px;
+		color:black;	
+	}
+	.big-news{
+		max-width: 1000px;
+		margin: 0 auto;
+		display:flex;
+		flex-direction: row;
+		justify-content: flex-start;
+		padding: 60px;
+	 }
+	 .navbar_tablet{
+		display:none; 
+	 }
+	 .news{
+		padding: 1rem;
+		width:650px;
+		height:90%;	
+	  }	
+	  html { font-size: 18px; 
+	  }
+	  
+}
+/* shake images*/
+@keyframes shake {
+	0% { transform: translate(1px, 1px) rotate(0deg); }
+	10% { transform: translate(-1px, -2px) rotate(-1deg); }
+	20% { transform: translate(-3px, 0px) rotate(1deg); }
+	30% { transform: translate(3px, 2px) rotate(0deg); }
+	40% { transform: translate(1px, -1px) rotate(1deg); }
+	50% { transform: translate(-1px, 2px) rotate(-1deg); }
+	60% { transform: translate(-3px, 1px) rotate(0deg); }
+	70% { transform: translate(3px, 1px) rotate(-1deg); }
+	80% { transform: translate(-1px, -1px) rotate(1deg); }
+	90% { transform: translate(1px, 2px) rotate(0deg); }
+	100% { transform: translate(1px, -2px) rotate(-1deg); }
+  }
+  
+  .card:hover {
+	animation: shake 0.5s;
+	animation-iteration-count: infinite;
+  }
+/** polariod photos **/
+.card {
+	position: relative;
+	width: 220px;
+	padding:10px;
+  } 
+   
+  .card img {
+	border: 10px solid #fff;
+	border-bottom: 45px solid #fff;
+	-webkit-box-shadow: 3px 3px 3px #777;
+	   -moz-box-shadow: 3px 3px 3px #777;
+			box-shadow: 3px 3px 3px #777;
```

Comment:
It's a great plus that you remembered about other browsers!
But what happened with indents over here?

Reply:
Fixed, indents.